conf.yaml

```diff
@@ -1060,15 +1060,15 @@ contents:
                 private:    1
                 single:     1
                 current:    *stackcurrent
-                branches:   [ {main: master}, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1 ]
-                live:       [ main, 8.5 ]
-                index:      client-docs/enterprise-search-node/index.asciidoc
+                branches:   [ {main: master}, 8.6]
+                live:       [ main, 8.6 ]
```

Comment:
I think we can use this, and then this value won't have to updated for each release:

```suggestion
                live:       *stacklivemain
```

Reply:
```18:44:50 INFO:build_docs:Live branch(es) <7.17> not in <branches> in book <Enterprise Search Node.js client> at lib/ES/Book.pm line 133.```


@chriscressman Am I right that this change is root of that failure? 😅

Reply:
I'll revert to get CI running again and check.

Reply:
Yeah, seems right. I thought `live` could contain "extra" branches, but apparently not.

Reply:
Getting there one build error at a time 👿

Reply:
Need to change filenames to `index.asciidoc` in:

- https://github.com/elastic/enterprise-search-js/pull/22
- https://github.com/elastic/enterprise-search-js/pull/23